src/main/java/edu/harvard/iq/dataverse/engine/command/impl/AbstractCreateDatasetCommand.java

```diff
@@ -119,7 +120,20 @@ public Dataset execute(CommandContext ctxt) throws CommandException {
         
         // Attempt the registration if importing dataset through the API, or the app (but not harvest)
         handlePid(theDataset, ctxt);
-        
+
+        DatasetType defaultDatasetType = ctxt.datasetTypes().getByName(DatasetType.DEFAULT_DATASET_TYPE);
```

Comment:
I wondered about that. Yes, it's currently possible to shoot yourself in the foot by deleting the "dataset" type. Sure, I could put something in to prevent this.

Reply:
Fixed in eb20155779.